source-stripe/tests/snapshots/capture.stdout.json

```diff
@@ -0,0 +1,5893 @@
+[
```

Comment:
There are several duplicated snapshot outputs for these tests. There should only be one snapshot output file for each of the capture/discover/spec commands, and they should be in the canonical file structure.